### DISCUSSION TITLE
How to hide the laravel car & spider?

### DISCUSSION THREAD

**Discussion by ziming** [api-platform/api-platform#2781] [Q&A]
I personally don't mind it, but people above me want the car & spider gone, so how do I hide it?

![CleanShot 2024-10-12 at 21 47 26](https://github.com/user-attachments/assets/ce49d30f-0e9e-45be-b7d8-068b53a0174c)

Thank you in advance

**Comment by beejaz** [comment]
https://api-platform.com/docs/core/configuration/#configuration

```
# Set this to false if you want Webby to disappear.
    show_webby: false
```

**Reply by soyuka** [reply]
mhh wondering if this is available inside Laravel..

**Reply by Deuchnord** [reply]
No, it's not. I'm preparing a PR to make it available.

**Comment by vinceAmstoutz** [comment]
Solved in https://github.com/api-platform/core/pull/6741
